knimin/lib/data_access.py

```diff
@@ -2399,6 +2399,178 @@ def getProjectNames(self):
         sql = """SELECT project FROM project"""
         return [x[0] for x in self._con.execute_fetchall(sql)]
 
+    def create_sample(self, sample_ids):
+        """Creates samples by ID
+
+        Parameters
+        ----------
+        sample_ids : list of str
+            Sample IDs to create
+
+        Returns
+        -------
+        bool
+            True if successful
+        """
+        sql = """INSERT INTO pm.sample (sample_id) VALUES (%s)"""
+        self._con.executemany(sql, [[x] for x in sample_ids])
+        return True
+
+    def delete_sample(self, sample_ids):
+        """Delete samples by ID
+
+        Parameters
+        ----------
+        sample_ids : list of str
+            Sample IDs to delete
+
+        Returns
+        -------
+        bool
+            True if successful
+        """
+        sql = """DELETE FROM pm.sample WHERE sample_id = %s"""
+        self._con.executemany(sql, [[x] for x in sample_ids])
+        return True
+
+    def create_sample_plate(self, sample_plate_info):
+        """Creates a new sample plate and sets values for mandatory fields
+
+        Parameters
+        ----------
+        sample_plate_info: tuple of (str, str, datetime, str, str)
+            name, email, creation_timestamp, notes, plate_type
+
+        Returns
+        -------
+        int
+            ID of the created sample plate
+        """
+        sql = """INSERT INTO pm.sample_plate (name, email, creation_timestamp,
```

Comment:
Can you update the SQL queries to follow the SQL coding guidelines exposed in [Qiita's contributing file](https://github.com/biocore/qiita/blob/master/CONTRIBUTING.md#sql-coding-guidelines)? Points 2 and 3 are the ones that apply here. As an example, this should read something like:
```python
sql = """INSERT INTO pm.sample_plate (name, email, creation_timestamp,
                                      notes, plate_type_id)
         VALUES (%s, %s, %s, %s, (SELECT plate_type_id
                                  FROM pm.plate_type
                                  WHERE name = %s))
         RETURNING sample_plate_id"""
```